docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+
+services:
+  hacktoberfest:
+    build: .
+    environment:
+      GITHUB_TOKEN: YOUR_TOKEN
+      NODE_ENV: production
+      GA_CODE: YOUR_CODE
```

Comment:
Could you load in the env file direct? https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option

Reply:
Done, added the use of the .env